[nnc] Disabled test case for failure in implementing conv1d

bertmaher opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#54756 [nnc] Disabled test case for failure in implementing conv1d**
* #54755 [nnc] Test case for computeAt with reduction

We have multiple bugs here, one relating to index flattening and the
other to computeAt.

Differential Revision: [D27354082](https://our.internmc.facebook.com/intern/diff/D27354082/)

bertmaher committed:
[nnc] Disabled test case for failure in implementing conv1d

We have multiple bugs here, one relating to index flattening and the
other to computeAt.

Differential Revision: [D27354082](https://our.internmc.facebook.com/intern/diff/D27354082/)

[ghstack-poisoned]

ZolotukhinM reviewed (approved):
Thanks for the test! Might be a good idea to file an issue for this too.

bertmaher commented:
Indeed I have!  #54757

bertmaher committed:
Add eliminateDeadStores on "[nnc] Disabled test case for failure in implementing conv1d"


We have multiple bugs here, one relating to index flattening and the
other to computeAt.

Differential Revision: [D27354082](https://our.internmc.facebook.com/intern/diff/D27354082/)

[ghstack-poisoned]

bertmaher committed:
Add eliminateDeadStores on "[nnc] Disabled test case for failure in implementing conv1d"


We have multiple bugs here, one relating to index flattening and the
other to computeAt.

Differential Revision: [D27354082](https://our.internmc.facebook.com/intern/diff/D27354082/)

[ghstack-poisoned]
